README.md

```diff
@@ -138,6 +140,26 @@ const plugin = createPlugin({
 
 * `[captureHealthCheckQueries = false]` Enable capture of timings for a [Health Check query](https://www.apollographql.com/docs/federation/api/apollo-gateway/#servicehealthcheck) received from an Apollo Federated Gateway Server.
 
+There are two configuration options for providing a callback that can be used to define custom attributes. The `customResolverAttributes` callback gets passed a `{ source, args, contextValue, info }` object [corresponding to attributes given to a resolver]() in Apollo server. The `customOperationAttributes` callback gets passed a `context` argument. These arguments can be given to imbue more details into custom attributes. Both callbacks must return an object that will be used to define custom attribute key-value pairs. For example,
+
+```js
+const plugin = createPlugin({
+  customResolverAttributes({ source, args, contextValue, info }) {
+    return {
+      allArgs: Object.keys(args).join(','),
+      returnType: info.returnType.name,
+      sourceBranch: source?.branch
+      stage: contextValue.event.requestContext.stage
+    }
+  },
+  customOperationAttributes(context) {
```

Comment:
Okay, polished the docs a bit more to point to the relevant information.

I used different parts of the docs because I didn't want to point our users to plugin documentation, as that seems like a more internal detail of how plugins are written.